evap/results/tests/test_views.py

```diff
@@ -52,6 +52,19 @@ def test_multiple_evaluations_per_course(self):
         self.assertNotContains(page, evaluation2.full_name)
         caches['results'].clear()
 
+
+    @patch('evap.evaluation.models.Evaluation.can_be_seen_by', new=(lambda self, user: True))
+    def test_order(self):
+        student = baker.make(UserProfile, email="student@institution.example.com")
+
+        course = baker.make(Course)
+        evaluation2 = baker.make(Evaluation, name_de='random_evaluation_b', name_en='random_evaluation_b', course=course, state='published')
+        evaluation1 = baker.make(Evaluation, name_de='random_evaluation_a', name_en='random_evaluation_a', course=course, state='published')
```

Comment:
You currently set `name_de`, which is never used as far as I see. I guess this was initially to switch the language and then check that their order changes? However, below, you only assert the order for the English page, not for the German one.

I guess it would probably be cool if you added the second check (the names would need to be so different that the ordering changes). Right now, this has a 50% chance of succeeding if the output order was random.

Reply:
Sounds reasonable, I changed it in the latest commit